roofit/hs3/inc/RooFitHS3/RooJSONFactoryWSTool.h

```diff
@@ -140,8 +140,10 @@ class RooJSONFactoryWSTool {
    std::string exportYMLtoString();
    bool importJSONfromString(const std::string &s);
    bool importYMLfromString(const std::string &s);
+   void importFunctionfromString(const std::string &jsonString);
+   void importVarfromString(const std::string &jsonString);
 
-   void importFunction(const RooFit::Detail::JSONNode &n, bool isPdf);
+   void importFunction(const RooFit::Detail::JSONNode &n, bool importAllDependants);
```

Comment:
```suggestion
   void importFunction(const RooFit::Detail::JSONNode &n, bool importAllDependants);
   void importFunction(const std::string &jsonString, bool importAllDependants);
```
At this point, your `importFunctionfromString()` is just a wrapper around `importFunction()`, only that it takes an `std::string` that will first be converted to a JSONNode.

Therefore, it is a good usecase for overloading instead of creating a new function with a new name. So I would:
* Rename `importFunctionfromString()` to just `importFunction()` and put its declaration right below the declaration of the original `importFunction()`
* also take the `importAllDependants` flag to be consistent with the two overloads
* Then, in the python code, you call `ROOT.RooJSONFactoryWSTool(self).importFunction(json_string, False)`